docs/getting-started/first-project.md

```diff
@@ -0,0 +1,327 @@
+---
+id: first-project
+title: Your First Project
+---
+
+Welcome to SCSS Starter.
+
+This tutorial introduces you to essentials of SCSS Starter by walking you through building a simple card.
+
+> New To Web Development?<br><br>
+> There are many resources to complement the SCSS Starter docs. Mozilla's MDN docs includes [css](https://developer.mozilla.org/en-US/docs/Web/CSS) introductions. Get Bem docs includes [bem](http://getbem.com/introduction/) introductions.
+
+## Setup
+
+To get started with SCSS Starter please read the [download](download.md) documentation. In order to start, please copy the `scss` folder into your own development environment.
+
+> The methods we mention in this documentation are based on Atölye15's own experiences over the years. These methods are rather obligations but more of suggestions. You may follow your own ways.
+
+## Setting Up Global Variables
+
+You need to set your global variables prior to starting the project based on your design's needs.
+
+### Colors
+
+> Please refer to [colors](core/colors.md) page for further explanation.
+
+In order to create your card component you should start with adding the colors to your `abstracts/_colors.scss` file.
+
+1. Set the primary color variable.
+
+```scss
+$color-palette-primary: (
+  50: #e8eaf6,
+  100: #c5cae9,
+  200: #9fa8da,
+  300: #7986cb,
+  400: #5c6bc0,
+  500: #3f51b5,
+  600: #3949ab,
+  700: #303f9f,
+  800: #283593,
+  900: #1a237e,
+);
+```
+
+2. To make the newly set color palette avaiable you need to use `color-palette` function. For that add `$color-palette-primary` map variable into `$g-color-palettes` map variable.
```

Comment:
```suggestion
2. To make the newly set color palette available, you need to use `color-palette` function. For that add `$color-palette-primary` map variable into `$g-color-palettes` map variable.
```

Reply:
Burada ayrica `for that` kullanimi da yanlis sanirim. o cumleyi yeniden formule etmek ya da o kullanimi degistirmek gerekebilir.